database/settings/2023_11_17_145037_remove_registration_links_to_general_settings.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use Spatie\LaravelSettings\Migrations\SettingsMigration;
+
+return new class extends SettingsMigration
+{
+    public function up(): void
+    {
+        $this->migrator->delete('general.individual_orientation');
+        $this->migrator->delete('general.org_orientation');
+        $this->migrator->delete('general.fro_orientation');
+        $this->migrator->delete('general.ac_application');
+        $this->migrator->delete('general.cc_application');
+    }
```

Comment:
In general this would be a dangerous approach because it would remove the existing settings. I think at this point it should be safe enough as the site hasn't gone live yet.